pages/auth/redirect.vue

```diff
@@ -23,8 +31,22 @@ onMounted(async () => {
       method: method as string,
       params: { code }
     })
+
+    if (signer.value) {
```

Comment:
What does it mean for `signer.value` to have no value? Should handle like error case

Reply:
[🥅 Handle missing signer issue](https://github.com/likecoin/nft-book-press/pull/175/commits/ef25d8ea1b16af5de132a12f4ac683c5126fa1c8)